CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.76.2
```

Comment:
Super NIT for future – this isn't a patch release, it's a minor version change.

Only quick bug fixes or small typos should go in as a patch release, not feature/theming additions.